pghoard/pghoard.py

```diff
@@ -126,14 +136,26 @@ def create_basebackup(self, cluster, connection_string, basebackup_path, callbac
             "--dbname", connection_string,
             "--format", "tar",
             "--xlog",
-            "--pgdata", raw_basebackup_path,
             "--progress",
             "--label", "initial_base_backup",
             "--verbose",
             ]
-        thread = PGBaseBackup(command, raw_basebackup_path, self.compression_queue, callback_queue)
+        if self.config["backup_sites"][site].get("multiple_tablespaces") is False:
+            basebackup_path = final_basebackup_path
+            command.append("--pgdata=-")  # special meaning, output to stdout
```

Comment:
`--pgdata=-` and `--pgdata -` probably do the same thing?  in that case it'd be nice to use the same way to format arguments for both variants.